sdk/synapse/Azure.Analytics.Synapse.Artifacts/api/Azure.Analytics.Synapse.Artifacts.netstandard2.0.cs

```diff
@@ -908,7 +1069,7 @@ public partial class AvroSink : Azure.Analytics.Synapse.Artifacts.Models.CopySin
     public partial class AvroSource : Azure.Analytics.Synapse.Artifacts.Models.CopySource
     {
         public AvroSource() { }
-        public System.Collections.Generic.IList<Azure.Analytics.Synapse.Artifacts.Models.AdditionalColumns> AdditionalColumns { get { throw null; } }
+        public object AdditionalColumns { get { throw null; } set { } }
```

Comment:
Is the change to `object` a generator change?

Reply:
@AlexanderSher , could you help us understand what changed in the generator to turn this into an `object` rather than a `List<AdditionalColumns>` ?

Reply:
In this particular case, property has become an object: 
- 2019-06-01-preview: https://github.com/Azure/azure-rest-api-specs/blob/88e7838a09868a51de3894114355c75929847a46/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Pipeline.json#L1187
- 2020-12-01-preview: https://github.com/Azure/azure-rest-api-specs/blob/bee724836ffdeb5458274037dc75f4d43576b5e3/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Pipeline.json#L1191

Reply:
Thank you, @AlexanderSher !  So, since this change doesn't include any changes to the generator, we can assume all changes are due to the swagger files and intended by the service team.